ktor-features/ktor-velocity/jvm/src/io/ktor/velocity/VelocityTools.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.velocity
+
+import io.ktor.application.*
+import io.ktor.response.*
+import io.ktor.util.*
+import org.apache.velocity.app.*
+import org.apache.velocity.tools.*
+import org.apache.velocity.tools.config.*
+
+private val ENGINE_CONFIG_KEY = "ENGINE_CONFIG"
+
+public fun EasyFactoryConfiguration.engine(configure : VelocityEngine.() -> Unit) {
+    data(ENGINE_CONFIG_KEY, configure)
+}
+
+/**
+ * VelocityTools ktor feature. Populates model with standard Velocity tools.
+ */
+public class VelocityTools(private val toolManager: ToolManager) {
```

Comment:
Can you make constructor internal, please?

Reply:
Oh, I made it private. I guess it's even better, no?